tests/basic/fsgsbase_unset/Makefile

```diff
@@ -0,0 +1,59 @@
+include ../../common.mk
+
+PROG=/bin/echo Hello World
```

Comment:
Hmm, wouldn't it be better to run something that actually uses TLS?

Reply:
I guess so, I don't know what a minimal test for TLS that reliably fails if FSGSBASE emulation is not working would look like though.